index.bs

```diff
@@ -5888,7 +5888,9 @@ The attestation certificate MUST have the following fields/extensions:
 
 - If the related attestation root certificate is used for multiple authenticator models, the Extension OID
     `1.3.6.1.4.1.45724.1.1.4` (`id-fido-gen-ce-aaguid`) MUST be present, containing the AAGUID as a 16-byte OCTET STRING.
-    The extension MUST NOT be marked as critical.
+    The extension MUST NOT be marked as critical. As [=Relying Parties=] may not know if the attestation root
+    certificate is used for multiple authenticator models, it is suggested that [=Relying Parties=] check if the extension
+    is present, and if it is, then validate that it contains that same AAGUID as presented in the [=attestation object=].
```

Comment:
I would make this its own paragraph, to reflect the separation that the first paragraph is the general requirement on authenticators and the second paragraph is implementation guidance for RPs.

Also, there is an `[RPS]` alias we can use.

```suggestion
    The extension MUST NOT be marked as critical.

    As [=[RPS]=] may not know if the attestation root
    certificate is used for multiple authenticator models, it is suggested that [=[RPS]=] check if the extension
    is present, and if it is, then validate that it contains that same AAGUID as presented in the [=attestation object=].
```